specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2023-08-01-preview/redisenterprise.json

```diff
@@ -1382,7 +1382,9 @@
         "Canceled",
         "Creating",
         "Updating",
-        "Deleting"
+        "Deleting",
+        "Scaling",
```

Comment:
As this will be breaking change for the customers who are using this preview API version already, please get breaking change review approval. See PR summary for breaking changes review process.

Reply:
@visingla-ms Will do re: breaking change approval.  Just FYI, this preview API version has not been released to customers yet so no one is using the changes.  I forgot to add the statuses for the original PR merged a week ago.

Reply:
If there are no customers using this API version, it is fine then.